src/libraries/System.Private.CoreLib/src/System/Numerics/Vector4.cs

```diff
@@ -209,11 +209,7 @@ public static Vector4 Clamp(Vector4 value1, Vector4 min, Vector4 max)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static Vector4 Lerp(Vector4 value1, Vector4 value2, float amount)
```

Comment:
The same be done for `Vector2`/`Vector3`

Reply:
It turns out `Vector3` was already optimized.